sonar-dotnet-shared-library/src/test/resources/RazorProtobufImporter/Cases.razor

```diff
@@ -0,0 +1,22 @@
+﻿@page "/cases"
```

Comment:
Can we add a comment here to explain that:
- the number of classes is 1 because Cases becomes a class deriving from `Microsoft.AspNetCore.Components.ComponentBase`
- the number of functions is 4 because, on top of `Increment.get`, `Increment.set` and `IncrementCount`, we also have `BuildRenderTree`

A question concerning the number of statements (15) (question which maybe goes beyond the scope of this PR): do we consider it correct?

Basically 15 = 13 of `BuildRenderTree` + 2 of `IncrementCount`:

```cs
protected override void BuildRenderTree(global::Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder __builder)
        {
            __builder.OpenElement(0, "p"); // 1
            __builder.AddContent(1, "Current count: "); // 2
#nullable restore
#line (3,20)-(3,32) 24 "C:\dev\TestApp1\RazorClassLibrary1\Test1.razor"
__builder.AddContent(2, currentCount); // 3

#line default
#line hidden
#nullable disable
            __builder.CloseElement(); // 4
            __builder.AddMarkupContent(3, "\n\n"); // 5
#nullable restore
#line (5,2)-(5,14) 24 "C:\dev\TestApp1\RazorClassLibrary1\Test1.razor"
__builder.AddContent(4, currentCount); // 6

#line default
#line hidden
#nullable disable
            __builder.AddMarkupContent(5, "\n\n"); // 7
            __builder.OpenElement(6, "button"); // 8
            __builder.AddAttribute(7, "onclick", global::Microsoft.AspNetCore.Components.EventCallback.Factory.Create<global::Microsoft.AspNetCore.Components.Web.MouseEventArgs>(this, 
#nullable restore
#line 7 "C:\dev\TestApp1\RazorClassLibrary1\Test1.razor"
                  IncrementCount

#line default
#line hidden
#nullable disable
            )); // 9
            __builder.AddContent(8, "Increment"); // 10
            __builder.CloseElement(); // 11
            __builder.AddMarkupContent(9, "\n\n"); // 12
#nullable restore
#line (9,2)-(9,17) 25 "C:\dev\TestApp1\RazorClassLibrary1\Test1.razor"
__builder.AddContent(10, IncrementAmount); // 13

#line default
#line hidden
#nullable disable
        }

...

    private void IncrementCount()
    {
        currentCount += IncrementAmount; // 14
        IncrementAmount += 1; // 15
    }
```

Reply:
This should be handled by https://github.com/SonarSource/sonar-dotnet/issues/7936